content/blog/2021/07/ApacheCon-Asia-2021/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "ApacheCon Asia Integration Track schedules"
+date: 2021-07-10
+authors: ["WillemJiang"]
+categories: ["Events", "Apache"]
+preview: "The integration track introduction for ApacheCon Asia2021"
+---
+[ApacheCon Asia 2021](https://apachecon.com/acasia2021/) will be held online from 6 August to 8 August. As a track chair of [the integration track](https://apachecon.com/acasia2021/tracks/integration.html), I will be presenting the topics related to integration and [the conference registration](https://hopin.com/events/apachecon-asia-2021) is now open.
+
+Integration is the method and technology of integrating heterogeneous systems built on different platforms and with different solutions. The most famous application integration project at the ASF is [Apache Camel](/), which has a variety of projects providing a variety of integration solutions. This time, the [ApacheCon Asia 2021](https://apachecon.com/acasia2021/) integration track brings you the latest information on the Apache Camel project, so follow me now and take a peek!
+
+For those of you who don't know much about [Apache Camel](/) yet, I suggest you check out [Claus](https://twitter.com/davsclaus) and [Andrea](https://twitter.com/oscerd2)'s presentation on [Apache Camel 3: The next generation of enterprise integration](https://apachecon.com/acasia2021/zh/sessions/1071.html), [Claus](https://twitter.com/davsclaus) and Andrea](https://twitter.com/oscerd2) are the lead developers on Apache Camel and [Claus](https://twitter.com/davsclaus) is the author of [Camel in Action](https://www.manning.com/books/camel-in-action-second-edition?). They led the development of Apache Camel 3, so in this talk they will present many of the latest developments on Camel 3, as well as the new members of the Camel family:
+
+* [Camel K](/camel-k/latest/) is a serverless integration platform that can be deployed on top of K8S, enabling low-code/no-code functionality and the ability to quickly stitch together large numbers of Camel connectors with the power of integration patterns.
+* [Camel Quarkus](/camel-quarkus/latest/) organically integrates [Apache Camel](/manual/latest/index.html) integration capabilities with its [component library](/components/3.10.x/index.html) and [Quarkus](https://quarkus.io/)  runtime, so that we can enjoy the performance gains of application loading, [developer fun](https://quarkus.io/vision/developer-joy) and [container-first](https://quarkus.io/vision/container-first) benefits that [Quarkus](https://quarkus.io/) provides while doing application integration development.
```

Comment:
```suggestion
* [Camel Quarkus](/camel-quarkus/latest/) organically integrates [Apache Camel](/manual/latest/index.html) integration capabilities with its [component library](/components/latest/index.html) and [Quarkus](https://quarkus.io/)  runtime, so that we can enjoy the performance gains of application loading, [developer fun](https://quarkus.io/vision/developer-joy) and [container-first](https://quarkus.io/vision/container-first) benefits that [Quarkus](https://quarkus.io/) provides while doing application integration development.
```